api/camunda/generation-strategy.js

```diff
@@ -0,0 +1,48 @@
+const removeDuplicateVersionBadge = require("../remove-duplicate-version-badge");
+const replace = require("replace-in-file");
+const outputDir = "docs/apis-tools/camunda-api-rest/specifications";
+const specFile = "api/camunda/camunda-openapi.yaml";
+
+function preGenerateDocs() {
+  hackChangesetDescription();
+}
+
+function postGenerateDocs() {
+  removeDuplicateVersionBadge(`${outputDir}/camunda-rest-api.info.mdx`);
```

Comment:
```suggestion
  removeDuplicateVersionBadge(`${outputDir}/camunda-8-rest-api.info.mdx`);
```

I believe that this would clean up the double-badge issue that @akeller mentions in https://github.com/camunda/camunda-docs/pull/3919#discussion_r1643309932. (It doesn't currently match the file name.)

Reply:
Good point, yeah, I'll fix that 👍